### ISSUE TITLE
DLC Pull-Down Related Tests

### ISSUE THREAD

**Issue by nkohen** [bitcoin-s/bitcoin-s#2613]
Replaces #2339

After #2339 and #2543 got into master pulling work down from the `adaptor-dlc` branch, some outstanding tests would be nice to have:

- ECDigitalSignature.fromFrontOfBytes[WithSigHash]
- SatoshisPerVirtualByte.toSatoshisPerKW
- RawTxFinalizers
- `DualFundingTxFinalizer.computeFees`
- sigFromMsgAndSigs
- CETCalculator multi-oracle functions (for more than 2 oracles)

A couple other things that may be nice to consider:

- Move local methods in `calculateOutcomeAndSig` out into private helper methods
- Separate responsibility of modifying pure function and evaluating ranges in `splitIntoRanges`
- `DLCOutcomeType` from `tlv` to `dlc` package
